### PR TITLE
Alert enhancement

### PR DESCRIPTION
**Pull request by aiwantaozi**
Changes:
Alert group and alert
Use wechat as a Notifier
expression based alert

Issue:
https://github.com/rancher/docs/issues/1166

### CODE REVIEW

File: content/rancher/v2.x/en/tools/notifiers-and-alerts/_index.md [hunk 1]
```diff
@@ -21,14 +21,15 @@ Rancher integrates with a variety of popular IT services, including:
...
+- **Wechat**: Send alert notifications to your Wechat at Work contacts.
 <br/>
 <br/>
 
 ### Adding Notifiers
 
 Set up a notifier so that you can begin configuring and sending alerts.
 
-1. From the **Global View**, open the cluster that you want to add a notifier to.
+1. From the **Global View**, open the cluster that you want to add a notifier.
```
Comment (loganhz) [inline content/rancher/v2.x/en/tools/notifiers-and-alerts/_index.md]: I think you don't need to remove `to`

**Comment by loganhz** [comment]
LGTM

File: content/rancher/v2.x/en/tools/notifiers-and-alerts/_index.md [hunk 2]
```diff
@@ -223,35 +306,44 @@ After you set up cluster alerts, you can manage each alert object. To manage ale
...
 
-1. Select the urgency level of the of alert. The options are:
+1. Select the urgency level of the alert. The options are:
 
     - **Critical**: Most urgent
     - **Warning**: Normal urgency
     - **Info**: Least urgent
 
-    Select the urgency level of the alert based on pod state and expendability. For example, an stateless pod that's not can be easily replaced, so select **Info**. However, if an important pod isn't scheduled, it may affect operations, so choose **Critical**.
+    Select the urgency level of the alert based on pod state and expandability. For example, a stateless pod that's not can be easily replaced, so select **Info**. However, if an important pod isn't scheduled, it may affect operations, so choose **Critical**.
```
[the remark this answers is not in the document]
Comment (loganhz) [reply content/rancher/v2.x/en/tools/notifiers-and-alerts/_index.md]: @aiwantaozi Please fix it

[the remark this answers is not in the document]
Comment (aiwantaozi) [reply content/rancher/v2.x/en/tools/notifiers-and-alerts/_index.md]: updated

**Pull request merged**